git/README.md

```diff
@@ -0,0 +1,45 @@
+# Git
+
+Git is pretty awesome. Here's some reasonable standards when it comes to using it.
+
+## Commit Messages
+
+At Mobify, we recommend a few standards regarding git commit messages:
+- If the commit is related to a JIRA ticket, prefix the case-sensitive ticket number to the start of the message
+  - The JIRA/GitHub integration is case-sensitive
+- Optionally, choose a [relevant emoji](#emoji-list) to categorize your commit
+- Finally, your main message content should follow the [git convention](http://git.kernel.org/cgit/git/git.git/tree/Documentation/SubmittingPatches#n117) of present-tense, imperative mood
+  - "...as if you are giving orders to the codebase to change its behaviour."
+
+E.g. `ABC-123: 🐛 Make feature X work by doing Y`
+
+## Emoji List
+
+Commit Type            | Emoji
+---                    | ---
+App/Astro              | 🚀 `:rocket:`
```

Comment:
Maybe emoji can be optional? I find them helpful for picking out a particular commit from a large list of them. If every commit message had a 🚀 , it wouldn't be very helpful. I would say the 🚀  might be more for a FEE/UI Dev making a change for Astro (in terms of styling or tooling), not the something the Astro team would use on each commit.